### PR TITLE
Show apply button for reviewing view

### PR DESCRIPTION
**Pull request by su-esther**
## Status:

:rocket: Ready

## Summary

Resource Popups currently show apply button. Only display the button when the organization is "reviewing", not "outreach".

Replaced EmailTag/PhoneNumberTag with IconTag => Can be deleted

## Changes
- Display tag in top-right corner
- Make location icon a uniform size
- Apply scroll for overflow
- Display organization's url. Click to open new tab with organization.
- Created IconTag for multi-purpose
- Apply check if organization.viewtype is "reviewing" to display the Apply button.

## Screenshots

Screen Shot 2023-04-27 at 12 32 47 AM

Screen Shot 2023-04-27 at 12 26 25 AM

### CODE REVIEW

File: src/components/icon-tag/types.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { IconType } from "components/icon";
+
+export interface LinkProps {
```
Comment (ayanmallik) [inline src/components/icon-tag/types.ts:3]: Let's rename this type to `IconTagProps` to be more comprehensive!

File: src/components/icon-tag/styles.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+import { TagStyleProps } from "./types";
+
+export const Text = styled.div`
```
Comment (ayanmallik) [inline src/components/icon-tag/styles.ts:4]: Nice job getting the text to wrap when it's too long! Could we enforce some padding/margin to wrap the content earlier and make sure it doesn't touch the end of the card/drawer?

image

**Review by ayanmallik — APPROVED** [review]
Awesome work Esther (and Jamie)! You killed it on this PR, given how clean and modular your code is. It's so cool to see how you were able to streamline all these icon tags into one reusable component - hope you see the value in how that consolidates our codebase tremendously!

After taking care of my proposed changes, feel free to also delete the deprecated `<EmailTag />` and `<PhoneNumberTag />` components!

**Comment by su-esther** [comment]
I saw that EmailTag was refactored recently to add href=`mailto:`. And a lot of other components use it which I couldn't render to see how my component looked on it. So I've left it alone for now!

**Pull request merged**